docs/workflows/blocks_bundling.md

```diff
@@ -108,6 +108,65 @@ REGISTERED_INITIALIZERS = {
 }
 ```
 
+## Serializers and deserializers for *Kinds*
```

Comment:
This feels like a really powerful and extensible pattern.  Wondering if it also opens the door for e.g. kind conversion/casting via serialize as one kind and deserialize as other

Reply:
yes as an opportunity to hook up this extension
no if you ask if this is possible without any changes to the PR
and probably no - if the question is broader - the polymorphism in *kinds* type system - that would need to be approached from different angle